src/channel/socketio-presence-channel.ts

```diff
@@ -41,7 +41,7 @@ export class SocketIoPresenceChannel extends SocketIoChannel implements Presence
      */
     leaving(callback): SocketIoPresenceChannel {
         this.on('presence:leaving', (member) => {
-            callback(member.user_info, this.channel);
+            callback(member.user_info, this.channel.socket);
```

Comment:
I'm just trying to send a socket instead of full channel object. Channel object includes list of subscribed events, and user probably dont need this information as was earlier (before this patch channels didnt have event list information). I just send socket info as was before.
![image](https://cloud.githubusercontent.com/assets/9549383/18255946/464125e8-73af-11e6-9d3f-25064005d462.png)